force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,63 +211,154 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
-        // Loop through each Opportunity name
-            // Create an Opportunity with the given name
+    // Look for account "accountName"; upsert if not there
+    Account account;
+    String stageName = 'Prospecting';
+    Date closeDate = Date.today().addMonths(3);
+
+    List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
```

Comment:
Same thing here. It would be better to call this `accs`. Developers are lazy. We like short names.